network/src/counters.rs

```diff
@@ -14,6 +14,29 @@ lazy_static::lazy_static! {
         // metric labels (dimensions)
         &["state", "role"]
     ).unwrap();
+
+    pub static ref RPC_REQUESTS: IntCounterVec = register_int_counter_vec!(
+        "libra_rpc_requests",
+        "Libra rpc requests counters",
+        &["unit", "state"]
+    ).unwrap();
+
+    pub static ref RPC_RESPONSES: IntCounterVec = register_int_counter_vec!(
+        "libra_rpc_responses",
+        "Libra rpc responses counters",
+        &["unit", "state"]
+    ).unwrap();
+
+    pub static ref LIBRA_RPC_LATENCY: Histogram = register_histogram!(
```

Comment:
I think we need to come up with consistent naming for these metrics. Some places you have the type named without the `LIBRA_` prefix where as you have it here. But in all cases you're actually naming the metric using `libra_`

Reply:
This one is because the RPC_LATENCY name is taken by the original counter. I'm planning to rename it later when I delete the old one. 
But yes I agree we should have a consistent naming for this. either with LIBRA prefix or not, currently I'm going with no LIBRA prefix so it's shorter, but it does feels like having the LIBRA prefix make more sense.

Reply:
I'm not advocating for either, just saying we should be consistent :)